Finjinon/PhotoCaptureViewController.swift

```diff
@@ -78,12 +82,13 @@ public class PhotoCaptureViewController: UIViewController, PhotoCollectionViewLa
         focusIndicatorView.alpha = 0.0
         previewView.addSubview(focusIndicatorView)
 
-        flashButton = UIButton(frame: CGRect(x: 12, y: 12, width: 70, height: 38))
+        flashButton = UIButton(frame: CGRect(x: buttonMargin, y: buttonMargin, width: 70, height: 38))
```

Comment:
Padding, or just margin is probably better since it's used horizontally and vertically. Or inset?